spec/support/idp_template.rb

```diff
@@ -5,7 +5,7 @@
 @include_subject_in_attributes = ENV.fetch('INCLUDE_SUBJECT_IN_ATTRIBUTES')
 @valid_destination = ENV.fetch('VALID_DESTINATION', "true")
 
-gem 'ruby-saml-idp', '~> 0.3.3'
+gem 'stub_saml_idp'
```

Comment:
So what changed here to support Rails 7?

Reply:
The blocking issue in the original gem was that `ruby-saml-idp` was using `unloadable` in the classes it defined, which is not supported in `zeitwerk`.  Ideally I would have submitted a PR to resolve this, but `ruby-saml-idp` hasn't been updated in over 5 years.

The other changes in `stub_saml_idp` were renaming changes `saml_idp` => `stub_saml_idp`, changing the way the test Rails apps were being generated to use template files, and some Rubocop lint changes.